src/main/java/uk/gov/hmcts/ccd/domain/casestate/EnablingConditionFormatter.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.hmcts.ccd.domain.casestate;
```

Comment:
if these new classes are meant to be generic, i.e. for working with enabling conditions on state, or in the future on events (we have a ticket for this feature proposal already) then better moving them outside the `casestate` package?

Reply:
this is new package anyway... we can rename the package if needed